ipaserver/install/ntpinstance.py

```diff
@@ -96,9 +82,6 @@ def __write_config(self):
                         break
                 if match:
                     srv_vals.remove(srv)
-                else:
```

Comment:
The list logic is now completely unncecessary. Please remove it and replace with a single boolean indicating if 127.127.1.0 was found or not.